Formula/rrdtool.rb

```diff
@@ -28,6 +28,11 @@ class Rrdtool < Formula
   patch :DATA
 
   def install
+    # fatal error: 'ruby/config.h' file not found
+    if MacOS.sdk && (MacOS.sdk.version != MacOS.version)
```

Comment:
Aside from `MacOS.version` I'm not sure these aren't what we'd consider to be internal API rather than something we want to use in formulae. Happy to defer to @MikeMcQuaid or @xu-cheng though.